pyproject.toml

```diff
@@ -17,6 +18,8 @@ types-protobuf = "^5.27.0.20240626"
 minio = "^7.2.8"
 gunicorn = "^23.0.0"
 pydantic-settings = "^2.6.0"
+tomli = "^2.1.0"
```

Comment:
added to deps, in order to parse toml at runtime and calculate super fast hashes with xxhash